scalaz/src/main/scala/org/scanamo/ops/ScalazInterpreter.scala

```diff
@@ -30,14 +30,21 @@ object ScalazInterpreter {
         }
         .asInstanceOf[Task[A]]
 
+    private[this] def catchAmazonDBException[A, A0](io: Task[A0]): Task[A] =
```

Comment:
@regiskuckaertz This function is basically identical to the `catchConditional` function above. I could not come up with a good way to keep the same behaviour of only catching `ConditionalCheckFailedException` exceptions for conditional actions whilst also catching `AmazonDynamoDBException` exceptions for get operations.